packages/search/lib/Documents.js

```diff
@@ -567,6 +569,8 @@ const prepublishScheduled = (elastic, redis, elasticDoc) => ({
       elasticDoc.meta.repoId,
       elasticDoc.id
     )
+
+    await createCache(redis).invalidate()
```

Comment:
so, this is not needed anymore, isn't it?

Reply:
Yes.